### ISSUE TITLE
Profiler mit Spy-On

### ISSUE THREAD

**Issue by linusha** [hpi-swa-teaching/InteractiveProfilingTool#21]

**Comment by linusha** [comment]
Estimate: 2h

**Comment by tillprochaska** [comment]
If I remember correctly, this *might* require some minor restructuring. Also, I’m unsure how the UI for running a single block would look like? So this story probably requires prior research into the existing UX and implementation. Estimate: 3-4h

**Comment by twollnik** [comment]
I agree with @tillprochaska that some UX research needs to be done prior to implementation. Depending on our findings, this could be rather complex, especially regarding the testing.

Estimate 5 h.
